### PR TITLE
feat: tabs component

### PR DESCRIPTION
**Pull request by trishaprile**
[![PR App][icn]][demo] | Fix RM-9809
:-------------------:|:----------:

## 🧰 Changes
- new built-in `<Tabs />` component for mdx

image

image

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: components/Tabs/style.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.TabGroup {
+  .nav {
```
Comment (rafegoldberg) [inline components/Tabs/style.scss:2]: Small detail: these child element classes should use BEM-style selectors, so `.TabGroup-nav` and `.TabGroup-tab` [per our SCSS selector conventions](https://next.readme.ninja/ui/#/Conventions/Writing%20CSS "Scroll to the bottom of this page.")!

**Review by rafegoldberg — CHANGES_REQUESTED** [review]

**Comment by kellyjosephprice** [comment]
Oh, you can merge this against `next` now!

**Review by rafegoldberg — APPROVED** [review]
Yay; thanks for the quick rewrite @trishaprile!

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v7.1.0</b>](/readmeio/markdown/releases/tag/v7.1.0)
<br>

**Pull request merged**